CHANGELOG.md

```diff
@@ -21,6 +21,9 @@
 
 ## Unreleased
 
+- bugfix: Graphite scaler should use latest datapoint returned, not earliest  (https://github.com/kedacore/keda/pull/2365)
+  
+
```

Comment:
Could you remove one white line, please?

Reply:
https://github.com/kedacore/keda/runs/4358667256?check_suite_focus=true

Reply:
Run golangci against the code...............................................Failed
- hook id: golangci-lint
- exit code: 4

ERRO Timeout exceeded: try increasing it by passing --timeout option

I note that the golangci-lint hook is timing out.


When I run this locally, it does pass, but its quite an intense check that it runs. My PR doesn't change much, so I'm a bit surprised that I might be the straw that broke the camel's back. Should I be adjusting timeouts as part of this PR?

Reply:
No need to do that, we need to fix this on a repo level. Thanks :)